src/nomadic/share/commands.py

```diff
@@ -0,0 +1,99 @@
+from collections import defaultdict
+from pathlib import Path
+
+import click
+
+from nomadic.util.rsync import (
+    print_rsync_summary,
+    rsync_minknow_data,
+    rsync_nomadic,
+    rsync_status,
+)
+from nomadic.util.workspace import Workspace, check_if_workspace
+
+
+@click.command(
+    short_help="Share summary nomadic workspace and associated minknow data to a cloud synchronized folder"
```

Comment:
This is the short help, better to keep it short like the other commands. Longer help goes in the doc string.